### ISSUE TITLE
A11y_AzureDataStudioPostgresExtension_Manage_Settings_DeveloperTool:The name for the format:Identifier case dropdown is defined as null.

### ISSUE THREAD

**Issue by tripathiakhil429** [microsoft/vscode#134559]
**GitHub Tags:**#A11yMAS;WCAG4.1.2;A11ySev2;#A11yTCS;#VisualStudioCode-Win32-Apr2021;#Benchmark;#Visual Studio Code Client;

### Repro Steps:
Tab till settings button and press enter.
Tab till dropdown.
Open developer tool using ctrl+shift+I and check if the name is defined for dropdown.

### Actual:
The name for the format:Identifier case dropdown is defined as null.

### Expected:
The name should be defined as the label for the dropdown for speech to text users to access it.

### User Impact:
Screen Reader users will get wrong information regarding the combo box.

**Comment by rzhao271** [comment]
I confirmed with NVDA that it would help if I added the settings id there.
The dropdown value is already read by the screenreader.
